src/pages/LaboratoryPage.tsx

```diff
@@ -1,37 +1,65 @@
-import { useParams, useNavigate } from 'react-router-dom';
-import { laboratories } from '../mocks/laboratories';
-import { Laboratory } from '../types/Laboratory.ts';
-import NavigationHeader from '../components/NavigationHeader/NavigationHeader';
+import { useParams } from 'react-router-dom';
 import LaboratoryPageContent from '@/components/LaboratoryPageComponents/LaboratoryPageContent/LaboratoryPageContent.tsx';
-import styles from '@styles/components/LaboratoryPage.module.scss';
-import ButtonBack from '@public/button_back.svg';
+import DefaultPageLayout from '@/components/DefaultPageLayout/DefaultPageLayout.tsx';
+import { useEffect, useState } from 'react';
+import { LaboratoryResponseData } from '@/DTO/LaboratoryDTO.ts';
+import { useLaboratoryProvider } from '@/providers/LaboratoryProvider/useLaboratoryProvider.ts';
 
 export default function LaboratoryPage() {
+  const [activeTab, setActiveTab] = useState('lab');
+  const [laboratory, setLaboratory] = useState<LaboratoryResponseData | null>(
+    null,
+  );
+  const [loading, setLoading] = useState<boolean>(true);
+  const [userError, setUserError] = useState<string | null>(null);
+
+  const laboratoryProvider = useLaboratoryProvider();
   const { id } = useParams<{ id: string }>();
-  const navigate = useNavigate();
-  const lab = laboratories.find((lab: Laboratory) => lab.id === Number(id));
 
-  if (!lab) {
+  useEffect(() => {
+    if (!id) {
+      setUserError('Id лабораторной не найден');
+      return;
+    }
+
+    setLoading(true);
+    laboratoryProvider
+      .getLaboratory(id)
+      .then(response => {
+        if (response.status === 200) {
+          setLaboratory(response.data);
+        }
+      })
```

Comment:
Если на странице больше нигде не планируется использовать провайдер, то можно память не занимать
```suggestion
    useLaboratoryProvider()
      .getLaboratory(id)
      .then(response => {
        if (response.status === 200) {
          setLaboratory(response.data);
        }
      })
```

Reply:
Как раз eslint говорит, что нельзя: "React Hook 'useLaboratoryProvider' cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function"

Reply:
Этот момент можно проигнорировать, так как useLaboratoryProvider не является хуком реакта.  Нужно будет мне добавить эту ошибку в игнор для eslint